UndertaleModLib/Models/UndertaleAnimationCurve.cs

```diff
@@ -141,6 +142,30 @@ public static uint UnserializeChildObjectCount(UndertaleReader reader)
         {
             reader.Position += 12;
 
+            // This check is partly duplicated from UndertaleChunks.cs, but it's necessary to handle embedded curves
+            // (For example, those in SEQN in the TS!Underswap v1.0 demo; see issue #1414)
+            if (!reader.undertaleData.IsVersionAtLeast(2, 3, 1))
+            {
+                long returnPosition = reader.AbsPosition;
+                uint numPoints = reader.ReadUInt32();
```

Comment:
what is a point here in this context?

Reply:
It's the list that's called "Points", AKA points on an animation curve asset

Reply:
I feel like them being named something like `animPoints` would be clearer. Or `animPointCount`.

Reply:
IMO itd be nice if this would be addressed

Reply:
Just refactored this a little bit - should be more clear now (and also technically with one less line of duplicated code)